app/views/devise/registrations/_common_info_fields.html.slim

```diff
@@ -1,31 +1,26 @@
 .form-group
   label.control-label.col-md-6.align for="First name" 
     | First name
-    span.required *
   .col-md-6
     = f.text_field :first_name, class:"form-control"
 .form-group
   label.control-label.col-md-6.align for="middle name" 
     | Middle name
-    span.required *
   .col-md-6
     = f.text_field :middle_name,class:"form-control"
```

Comment:
@VaibhaviJoshi-BTC : Use space after comma for better readability & do it all below places